### PR TITLE
Fix/make app run

### LINKED ISSUE #9: fix(analysis-platform): make app work again
a lot of changes has happened to the functionality and dmss and blueprints, which has caused the analysis platform app to break

### PR DESCRIPTION
**Pull request by KristianKjerstad**
## What does this pull request change?
make the analysis portal compatible with the latest versions of dm-core, dmss, dm-job etc.
Note: this PR will make the app start, but there are still some bugs - these will be fixed in a new PR.

main changes:
* update documents to use new format (alias CORE instead of system/SIMOS)
* move job_handlers folder to repo root folder
* update usage of components from dm-core npm package
## Why is this pull request needed?
make app work again.
## Issues related to this change
closes #9

### CODE REVIEW

File: src/plugins/analysis-platform/components/Layout/Content.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import { Layout } from 'antd'
 import styled from 'styled-components'
+import { ReactNode } from 'react'
 import { useParams } from 'react-router-dom'
 
-import { TContent, DocumentPath } from '@development-framework/dm-core'
+import { DocumentPath } from '@development-framework/dm-core'
```
Comment (KristianKjerstad) [inline src/plugins/analysis-platform/components/Layout/Content.tsx:7]: TContent has been removed from dm-core

File: src/plugins/analysis-platform/Routes.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
-import { TRoute } from '@development-framework/dm-core'
 import { AssetOverview } from './modules/Asset'
+import { ReactNode } from 'react'
 import { AssetCreate } from './modules/Asset'
 import { AnalysisCreate, AnalysisOverview } from './modules/Analysis'
 import { View } from './modules'
 
+type TRoute = {
```
Comment (KristianKjerstad) [inline src/plugins/analysis-platform/Routes.tsx:7]: TRoute has been removed from dm-core

File: src/plugins/analysis-platform/plugins/sima-application-input/src/EditSimaApplicationInput.tsx [hunk 3]
```diff
@@ -89,7 +91,7 @@ export const EditSimaApplicationInput = (props: IDmtUIPlugin) => {
     }
   }
 
-  if (loading || formData === null) return <Loading />
+  if (loading || document === null || formData === null) return <Loading />
```
Comment (KristianKjerstad) [inline src/plugins/analysis-platform/plugins/sima-application-input/src/EditSimaApplicationInput.tsx:94]: without checking `document === null`, typecsript compains further down . 

document === null and loading is kind of the same thing, so I am very open to other suggestions for how ot solve this.

File: src/plugins/analysis-platform/modules/Asset/AssetView.tsx [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
-export const AssetView = (props: IDmtUIPlugin): JSX.Element => {
-  const { documentId, dataSourceId } = props
-  const [asset, loading] = useDocument(dataSourceId, documentId)
+export const AssetView = (props: IUIPlugin): JSX.Element => {
+  const { idReference } = props
+  const [dataSourceId, documentId] = idReference.split('/', 2)
+  const [asset, loading] = useDocument<TAsset>(idReference)
 
-  if (loading) {
+  if (loading || asset === null) {
```
Comment (KristianKjerstad) [inline src/plugins/analysis-platform/modules/Asset/AssetView.tsx:12]: without checking `asset === null`, typecsript compains further down . 

asset === null and loading is kind of the same thing, so I am very open to other suggestions for how ot solve this.

File: src/plugins/analysis-platform/App.tsx [hunk 5]
```diff
@@ -7,11 +7,16 @@ import {
   ApplicationContext,
   Header,
   FSTreeProvider,
-  TLayout,
 } from '@development-framework/dm-core'
 import Content from './components/Layout/Content'
 import Menu from './components/Layout/Menu'
 
+type TLayout = {
```
Comment (KristianKjerstad) [inline src/plugins/analysis-platform/App.tsx:14]: TLayout has been removed from dm-core

**Review by soofstad — APPROVED** [review]
Good work!

**Pull request merged**